games-arcade/openclaw/openclaw-1.0.3.recipe

```diff
@@ -0,0 +1,67 @@
+SUMMARY="A 2D side-scroller platform video game"
+DESCRIPTION="Captain Claw is a 2D side-scroller platform video game \
+published by Monolith Productions in 1997 about the anthropomorphic \
+pirate cat Captain Nathaniel Joseph Claw, who sets on a quest to find \
+an ancient amulet while fighting enemies and solving puzzles."
+HOMEPAGE="https://www.lith.com/games/captain-claw"
+COPYRIGHT="2012 Michael L. McShaffry and David Graham"
+LICENSE="GNU GPL v3"
+REVISION="1"
+srcGitRev="150f7edb7542f3f877b42afc948cc24ea9a46c48"
+SOURCE_URI="https://github.com/pjasicek/OpenClaw/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="9521811f572ca6c61288628cc32ff86f9c7a455afa09387232282f936a531845"
+SOURCE_DIR="OpenClaw-$srcGitRev"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+PROVIDES="
+	openclaw$secondaryArchSuffix = $portVersion
+	app:openclaw = $portVersion
+	lib:libsdl2 = $portVersion
+	lib:libsdl2_mixer = $portVersion
+	lib:libsdl2_ttf = $portVersion
+	lib:libsdl2_image = $portVersion
+	lib:libsdl2_gfx = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libtinyxml$secondaryArchSuffix
+	lib:libsdl2$secondaryArchSuffix
+	lib:libsdl2_mixer$secondaryArchSuffix
+	lib:libsdl2_ttf$secondaryArchSuffix
+	lib:libsdl2_image$secondaryArchSuffix
+	lib:libsdl2_gfx$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libtinyxml$secondaryArchSuffix
+	devel:libsdl2$secondaryArchSuffix
+	devel:libsdl2_mixer$secondaryArchSuffix
+	devel:libsdl2_ttf$secondaryArchSuffix
+	devel:libsdl2_image$secondaryArchSuffix
+	devel:libsdl2_gfx$secondaryArchSuffix
+	"
+	
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	mkdir -p build
+	cd build
+	cmake .. -DDATAPATH='"$appsDir"'
+	make $jobArgs
+}
+	
+INSTALL()
+{
+	cd ..
+	mkdir -p $appsDir/OpenClaw
+	cp -r OpenClaw-$srcGitRev/* $appsDir/OpenClaw
```

Comment:
I'm pretty sure you don't need the entire source code of OpenClaw to run the game. Please copy only what's needed.

Also, since the game can only run from the command line in it's current state, I'd recommend installing it to `binDir` instead.